hazelcast-client-new/src/test/java/com/hazelcast/client/map/ClientMapUnboundReturnValuesTestSupport.java

```diff
@@ -178,6 +179,13 @@ private void fillToUpperLimit(IMap<Integer, Integer> fillMap, IMap<Integer, Inte
         assertEquals("Expected map size of client map to match upperLimit", upperLimit, queryMap.size());
     }
 
+    private void checkException(QueryResultSizeExceededException e) {
```

Comment:
Checks the exception if it has been thrown by a `QueryPartitionOperation`, which would mean that the query execution happened twice. This happened in some cases because the exception of the `QueryOperation` has not been caught correctly.